src/ripple/protocol/impl/STArray.cpp

```diff
@@ -32,13 +32,13 @@ STArray::STArray()
     //v_.reserve(reserveSize);
 }
 
-STArray::STArray (STArray&& other)
+STArray::STArray (STArray&& other) noexcept
     : STBase(other.getFName())
```

Comment:
`STBase` is not nothrow constructible from `other.getFName()`, but I believe it could be made so.

Reply:
Good point. Just made that possible in [this commit](https://github.com/ripple/rippled/pull/2551/commits/a4fc4d70bab422372568fb4248efab5318a9ece8). I appreciate the quick turn-around and excellent observations. I'll try to get to the other comments tonight or tomorrow.